infrastructure/templates/infrastructure/project_detail.html

```diff
@@ -217,6 +217,50 @@ <h3>Substation(s)</h3>
          </section>
       {% endif %}
 
+      {% if object.infrastructure_type.name == 'Pipeline' %}
+            <section class="related-pipeline-data">
+               <h2>Related Pipeline Data</h2>
+               <div class="row double">
+                  <section>
+                     <h3>Capacity</h3>
+                     <p>{% if object.project_capacity %}
+                        {{ object.pipeline_capacity_property }}
+                     {% else %}
+                        &mdash;
+                     {% endif %}
+                     </p>
+                  </section>
+                  <section>
+                     <h3>Estimated Throughput</h3>
+                     <p>
+                        {% if object.pipeline_throughput_property %}
+                           {{ object.pipeline_throughput_property }}
+                        {% else %}
+                           &mdash;
+                        {% endif %}
+                     </p>
+                  </section>
+               </div>
+               <div class="row double">
+                  <section>
+                     <h3>Fuel(s)</h3>
+                     <ul class="clean">
+                        {% for fuel in fuel_types %}
+                           <li>{{ fuel }}</li>
```

Comment:
I wonder if these are supposed to be categories.. hard to tell from the wireframe, but a generic **Oil** item seems broad.

Reply:
Also, I think we should add an empty case with a dash:

```
{% for ... %}
blah
{% empty %}
  <li>-</li>
{% endfor %}
```

Reply:
done

Reply:
Is it? I don't see this change.